### PR TITLE
Adding auto-restart devnet script for studio

### PR DESCRIPTION
**Pull request by evgenykuzyakov**

### CODE REVIEW

**Comment by evgenykuzyakov** [comment]
@janedegtiareva Looks like after I fixed kill_devnet.sh to actually kill it, the nearlib integration test started to fail. It might be a bug that we didn't kill the devnet before

File: scripts/auto_restart_devnet.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    sleep 3
+    echo "Deploying code at tests/hello.wasm"
+    scripts/rpc.py deploy -s $ACCOUNT $CONTRACT tests/hello.wasm
+    sleep 3
+    while :
+    do
+        VALUE=$(openssl rand -hex 10)
+        echo "Setting value $VALUE"
+        scripts/rpc.py schedule_function_call -s $ACCOUNT $CONTRACT setValue --args=\{\"value\":\"$VALUE\"\}
+        sleep 5
```
Comment (vgrichina) [inline scripts/auto_restart_devnet.sh]: I would be more conservative with delays and put them into overridable env vars

File: scripts/auto_restart_devnet.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+set -e
+
+pushd $(dirname "$0")/..
+
+trap ctrl_c INT
+
+function ctrl_c() {
+    echo "** Interrupted by CTRL-C"
+    DEVNET_PID=$(pgrep devnet)
```
Comment (vgrichina) [inline scripts/auto_restart_devnet.sh]: should we instead save pid when starting devnet?
https://unix.stackexchange.com/questions/30370/how-to-get-the-pid-of-the-last-executed-command-in-shell-script

Comment (evgenykuzyakov) [reply scripts/auto_restart_devnet.sh]: good point

File: scripts/auto_restart_devnet.sh (hunk 1 above)
Comment (evgenykuzyakov) [reply scripts/auto_restart_devnet.sh]: I think that delay of 5 sec is cool. Maybe we should increase some setup delays

**Comment by ilblackdragon** [comment]
@evgenykuzyakov progress here?

**Comment by evgenykuzyakov** [comment]
Don't need it anymore. It needs to update to `pynear`. We'll keep the branch if we need to bring it back.

**Pull request closed without merging**